lib/path-utils.js

```diff
@@ -0,0 +1,225 @@
+import path from "path"
+import { promises } from "fs"
+const { readFile, stat } = promises
+
+import isPath from "is-valid-path"
+
+import { unique } from "./utils"
+
+/** Converts given path to Posix (replacing \\ with /)
+ * @param {string} givenPath   Path to convert
+ * @returns {string}          Converted filepath
+ */
+export function posixifyPath(givenPath) {
+  return path.normalize(givenPath).replace(/\\/g, "/")
```

Comment:
I think this should work

```suggestion
  return path.posix.normalize(givenPath)
```

Reply:
According to this issue, these two are not equivilant
https://github.com/nodejs/node/issues/12298#issuecomment-292935749